pkg/skaffold/survey/survey_test.go

```diff
@@ -53,3 +57,118 @@ func TestDisplaySurveyForm(t *testing.T) {
 		})
 	}
 }
+
+func TestShouldDisplayPrompt(t *testing.T) {
+	tenDaysAgo := time.Now().AddDate(0, 0, -12).Format(time.RFC3339)
+	fiveDaysAgo := time.Now().AddDate(0, 0, -5).Format(time.RFC3339)
+	twoMonthsAgo := time.Now().AddDate(0, -2, -5).Format(time.RFC3339)
+	threeMonthsAgo := time.Now().AddDate(0, -3, -5).Format(time.RFC3339)
```

Comment:
The `-12` threw me for a loop.  `-10` works just like `-5` does below.
```suggestion
	tenDaysAgo := time.Now().AddDate(0, 0, -10).Format(time.RFC3339)
	fiveDaysAgo := time.Now().AddDate(0, 0, -5).Format(time.RFC3339)
	twoMonthsAgo := time.Now().AddDate(0, -2, -5).Format(time.RFC3339) // less than 90 days ago
	threeMonthsAgo := time.Now().AddDate(0, -3, -5).Format(time.RFC3339) // at least 90 days ago
```